### ISSUE TITLE
[Question]: Provide a latest puppeteer dependency

### ISSUE THREAD

**Issue by vigneshsivaji** [IBMa/equal-access#1553]
### Project

a11y checker

### Description

In accessibility-checker, the puppeteer dependency have older version ^13.0.0. can you update this version into latest version 20.8.2?

**Comment by shunguoy** [comment]
@vigneshsivaji  the engine dependency update is ongoing and should be released soon. thanx.

**Comment by tombrunet** [comment]
@vigneshsivaji Please try version [4.0.0-rc.1](https://www.npmjs.com/package/accessibility-checker/v/4.0.0-rc.1)

**Comment by vigneshsivaji** [comment]
thanks for the update
